### PR TITLE
pagination

### PR DESCRIPTION
**Pull request by cabilangan112**

### CODE REVIEW

File: jassen/project/blog/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      <p>{{index.category.title}}</p>
+      <p><li><a href='{% url "blog:post" index.id %}'   role="button">Detail</a></li></p>  
+       </div> 
+     {% endfor %} 
+  {% endfor %}
+
+  <div class="pagination">
+      <span class="step-links">
+          {% if index.has_previous %}
+              <a href="?page=1">&laquo; first</a>
```
Comment (rocity) [inline jassen/project/blog/templates/index.html]: Don't hardcode URLs in the Django template https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#url

File: jassen/project/blog/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+              <a href="?page={{ index.previous_page_number }}">previous</a>
+          {% endif %}
+
+          <span class="current">
+              Page {{ index.number }} of {{ index.paginator.num_pages }}.
+          </span>
+
+          {% if index.has_next %}
+              <a href="?page={{ index.next_page_number }}">next</a>
```
Comment (rocity) [inline jassen/project/blog/templates/index.html]: Don't hardcode URLs in the Django template

File: jassen/project/blog/urls.py [hunk 3]
```diff
@@ -2,6 +2,6 @@
 from . import views
 app_name = 'blog'
 urlpatterns = [
-    path('post/<int:post_id>/', views.PostView.as_view(), name='post'),
-]
- 
\ No newline at end of file
+    path('<int:pk>/', views.IndexView.as_view(), name='index'),
+    path('Post/<int:post_id>/', views.PostView.as_view(), name='post'),
```
Comment (rocity) [inline jassen/project/blog/urls.py]: avoid capitalization in your routes. good examples are:

```
http://example.com/blog/post/1/
http://example.com/user/profile/1/
http://example.com/article/business/how-to-do-urls/
```

File: jassen/project/blog/views.py [hunk 4]
```diff
@@ -4,15 +4,26 @@
...
+    def get(self, request, pk, *args, **kwargs):
+        index_list = Index.objects.all()
+        paginator = Paginator(index_list, 1)
+        page = request.GET.get('page')
+        index = paginator.get_page(page)
+        context = {'index':index,}
+        return render(request, "index.html", context)
+
+
+
```
Comment (rocity) [inline jassen/project/blog/views.py]: separate `Class`es with 2 blank lines. Remove any excess https://www.python.org/dev/peps/pep-0008/#blank-lines

Comment (cabilangan112) [reply jassen/project/blog/views.py]: ok sir

**Pull request merged**